internal/deviations/deviations.go

```diff
@@ -204,5 +204,5 @@ var (
 	SchedulerInputParamsUnsupported = flag.Bool("deviation_scheduler_input_params_unsupported", false, "Device does not support scheduler input parameters")
 
 	NetworkInstanceTableDeletionRequired = flag.Bool("deviation_network_instance_table_deletion_required", false,
-		"Set to true for device requiring explicit deletion of network-instance table, default is false")
+		"Set to true for device requiring explicit deletion of network-instance table, default is false. Full OpenConfig compliant devices should not set this deviation to true")
```

Comment:
@dplore what would be the expected behavior for a device which receives a delete request on a protocol table path only? e.g. /network-instances/network-instance/tables/table[protocol=BGP] is deleted, but BGP protocol is still configured and has established adjacencies?

Reply:
> @dplore what would be the expected behavior for a device that receives a delete request on a protocol table path only? e.g. /network-instances/network-instance/tables/table[protocol=BGP] is deleted, but BGP protocol is still configured and has established adjacencies? [it is not currently part of the test, but in a general case]

I would expect an error to be returned.   This is maybe not obvious because in the [use cases](https://www.openconfig.net/docs/models/network_instance_redist/) for tables, there is no scenario where a client would create or delete items from the tables list.   But please let me know if I am missing anything.

Reply:
Thanks @dplore 

>  But please let me know if I am missing anything.

I would agree, yes. The only reason I asked is that clarification about a fully compliant device -> "enabled" means that tables _are_ deleted by a client in this scenario (however, together with a protocol in this specific scenario); so I was thinking about what this delete operation would entail in a general case.